src/util.h

```diff
@@ -223,17 +224,35 @@ inline bool IsSwitchChar(char c)
 class ArgsManager
 {
 protected:
+    friend class ArgsManagerHelper;
+
     mutable CCriticalSection cs_args;
-    std::map<std::string, std::string> mapArgs;
-    std::map<std::string, std::vector<std::string>> mapMultiArgs;
-    std::unordered_set<std::string> m_negated_args;
+    std::map<std::string, std::vector<std::string>> m_override_args;
+    std::map<std::string, std::vector<std::string>> m_config_args;
+    std::string m_section;
```

Comment:
I think this variable could be named better (and commented). If I'm not mistaken, this is the network that the node has been configured to run on (mainnet, testnet or regtest). Would a better name be `m_network`?